spec/text-editor-spec.coffee

```diff
@@ -1986,6 +1986,82 @@ describe "TextEditor", ->
         expect(editor2.getSelectedBufferRanges()).not.toEqual editor.getSelectedBufferRanges()
 
   describe "buffer manipulation", ->
+    describe ".moveLineUp", ->
+      describe "when there is only one line selected", ->
+        it "moves the line up by one row", ->
+          editor.setSelectedBufferRange([[3, 2], [3, 9]])
+          expect(editor.getSelectedBufferRange()).toEqual {start: {row: 3, column: 2}, end: {row: 3, column: 9}}
```

Comment:
Feeling a bit like the boy who cried wolf because I didn't have any problems using the array syntax today.. strange. changes made